### DISCUSSION TITLE
Font is different for each device.

### DISCUSSION THREAD

**Discussion by UnbalancedSkunk** [getnikola/nikola#3781] [Q&A] [answered]
I couldnt solve this problem. Theme : Bnw i edited bnw....css and style.css

**Comment by Kwpolska** [comment, marked as answer]
It seems the theme is using http for its fonts, which browsers are likely to block. I updated the theme, you can reinstall and try again.

If the issue still occurs, please be more specific about what goes wrong.

**Comment by UnbalancedSkunk** [comment]
Thank you.
